### ISSUE TITLE
feat(body editing): Find & Replace

### ISSUE THREAD

**Issue by chriswhong** [qri-io/desktop#422]
Find and Replace will be useful for people cleaning datasets.  I can imagine find/replace with validation being a key part of dataset cleaning in desktop.

**Comment by Mr0grog** [comment]
> I can imagine find/replace with validation being a key part of dataset cleaning

Yes! During the call this morning, one of the smaller things I was thinking about was: could find/filter and validation use the same syntax? They’re ultimately doing a very similar thing.
